stories/guideline/docs-template.mdx

```diff
@@ -0,0 +1,43 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  Primary,
+  Controls,
+  Unstyled,
+  Source,
+} from '@storybook/blocks';
+
+{/* as SelectStories from './selectv2.stories'; */}
+
+<Meta title="Guideline" isTemplate />
+
+# Component name
+
+General description
+
+## Style
+
+## How it is used ?
```

Comment:
```suggestion
## How is it used ?
```

Reply:
In English, no space between the last letter and the question mark. Should be:
## How is it used?